sdk/iot/Azure.Iot.Hub.Service/src/Authentication/SharedAccessSignatureBuilder.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    /// <summary>
+    /// Builds the shared access signature based on the access policy passed.
+    /// </summary>
+    internal sealed class SharedAccessSignatureBuilder
+    {
+        public SharedAccessSignatureBuilder()
+        {
+            TimeToLive = TimeSpan.FromMinutes(5);
```

Comment:
comment about this value?

Reply:
this is actually not required, since the default value is set in the sas token provider. I've removed this.